documentation/src/docs/asciidoc/images/connect-prometheus-2.svg

```diff
@@ -0,0 +1,3 @@
AAAAAeAQCLAcy0jOypJsPrO8fOXkqzdmwWZXb1F5/Sbq4XOvSz/xs489o4cKFqttW7mXFodXu3bs98vvmRBnptogemh5hpAmhnq0lVNcdSEdbDPRxW5B8a+mnP/l3uv9b/6TL16Cx7Fnqrfmrzce+iPw21YfO8vo68JDCyKj+QbWItbGY5bBC1vnOn6j9jf8Y0u8099DqmZhGb9c1yb20nGExo2mjtCQVlxRfw6cKAAAAAAC4CwGWHRnpGdnSzevW93OtK655pUbuzDSoxmLtXMNq06ZNbve0Sg7tpcXDu+jr0UaKDLT/+ggMMlBQcLC0hFCgIYQCAgOl+4Lk+y0Zuzrl257uLjL1GKXbbum+DqfWpbwtiHY2hMq3G9aspEUqbXtDYWtmQjNfFHdnPJyQhxVaKykpoffef1+o/WU9WyHj+ljGs8c98vtD71kpB1oVvcG0r/aaw95ZXBOLa2OZX1rFJcXV+HQBO5+5SdJNktXdtu4z4559sU78av75RBXvmpPSYhn+8teWkyQUmu/H5AkAAACgdQiwbJ9I25xpMGXJMpq4dJkqt6nhzCk6+twzLv2M2sIr7m21detWl3/urphuWjGyi0YH274Q52AqJDySQiKjB4RTnsLhVldrM3W1tzp8HvfU+toPfkFhdyzRzevRXlF3M1+FWPaGFDY2Nsq1sTo7O4Xab3ffdRelp6ffaGM1Z6nluWyX6mMNhntnha/4CZ2Pm0D7aq/aLAh/esc2qnrnLfO3d0oX2YX4lFHdZ2KW8mUS3QiUzPdxiDQDe0k1iqSlmvpCsBN4PfrtNcWvo/XK68iTrx/u8crHdJ90bAuwpwEAQGsQYA08qciVbjZa36/m8MqdYYNqCq8OHDhAP9/0c6efPyq4l9aO6qSvRxntX5iHhFF47AivBVaD6e0xUmdzo8NAK3rtc7oJsnprDpKx7Bd2H/dViMVsDSlk//2Xv1BlZaVQ+826yDvrOryP2l57xuN/i2tnRazZTJ+Pnz6gZ1bFnt1Uuff6UF6EWCr+LARd4F5fPOvyPvTq8uhrisNefu/zZeDLgVYWjiMAAGgFAiwnTth5yCAPHVQjLYdXXEx75YqVTj2XhwLmxHc4DK04rIoYHicPCRSJydRLHY1XbYZZARHRFLvhj2RInKz516ejeljMlyGWvVkKRSzwzubOnUvzpMVS80+/RT01n3rngyU8iiLX/5r+HjeZCuub+/ZN/xBrNXoHqPPzEHTtnLTwzBoFqGnn8uspSbqp8uMqbJKOWS6OBAAAqB0CrEFO1tUcXrkz26BaCrb/+Ec/pqKiokGfx8MDnxjjuN4UB1aRI0dRQECg8NvN9bTarl4mU2//4Y7h936Hoh56SvOvU0f1sJgvQ6yRI0fSlKlTBtwv6pBCW72xhjJbobOCEiZR+6pNtNcwmj58408IsVT+mQhghT+Ic9GrctDXE/eA8vdQ2+EIHgEAQO0QYJE2wyt2eMN6aqqpdvr5a9asoTVr1wi9TTyz4KIHFg1aoD1nTAfdHdPtuPEHBlJU3Bi/DRMcCq6Z1XrlUr8gi3tjjcx/X77VKlNjJXV/8B2Hz/HV7IQsNCyU0jPSbT7GIZZosxQyW72x2rZuoK4j+73+t0PmLqI3x82lFzdef7vFcEKVfS4CDGIL9QVaCEr6v55EONnG+y0AAKie7gMsLRZsZ1azfw1qwcIFtHGj2NcqpaWl9Nj3HnP4nAdHdtGKuMF7voRGRlPYsOGqb7+drc3U0XS1330j8t6joJvGafY121OeTz2f/9nhc07F5FBb0FifrI/BYKCZqTP79Wwye++996jk+HHh9mFMTAx9d/XqfuvcW/8VNT/zLY8Webf7wRMeRZ9lPUL//uouDqVxUSXmZ2MuIcCCodkuLesRZiHAAgAA8Nh1hJ4DLOmEIlu6ed36frWHVxxccYDlrPj4eNr/5n6ht2mw8Gp6RA/98mbnhkpGjhxNhpBQl/5+i9FEH9Z3018vdtKJa8ZBnz8mLJBujzXQv4wPo5SoIK/uG66R1XzpQr/eWLEbtlPwlAzNvna7Dn6TqNtx0FI6/OfUExDus3WyV9z94/JyOnjwoJD78VtLl9LEiRP73de+85fU+e4On61Dx2130m+uxs549hfPfoyPZKE+H3MJARZ4BhcSX6/nIcMIsAAAADxDtwGWMi34B9b385BBHjqoVu4UbefwikMsUdXW1srDBu35dVIrTQjtdep3uRJeHa7vpi2ftdHFjl6PbMc/jw+lh5PCKcoQ4JX91NpwiYxdN+p9abknVm/th2Q89qTD53APLO6J5UvTb5tOMbExA+6vq6ujP7z+upD7MiEhgVY8+GD//evD3lhmQeNTesJXPHmn4dY5h/HRLMRnZC4hwALP215cUpytw9cTAiwAAAAPCNTjRiuzwQwIr0ZOnqrq8Ir97+YNLj2fi7aLHF6xFQ+usB0WRPTQwUnNTodXEcNvGjS84p5W2cebaG7hVXrqVIvHwiv2/8530n1HrslLZUuPx/cTF6I3hNwYEnZ1Axc8b9bmG1f8HfLi8Hj3XKAJrX/26XqVf1wuh1XWRo8eTTnr18uF1EVTU1NDL+blycXnr+/fuHEU88pRCr1npc/Wo+d8ZVDLfzzyYfOGJUbj6aPz8PEMoEmrOMxRAlIAAAAAl+iyB5Z04sT1GPp1kwiPu4nm521V9Xad3rGNqt55y+nnZ2Zm0gv/+YLQ27Rp0yY6eGDg8Ctna12ZDVbzioOrfz3RTJ+5GSzxzI1paWkUPzZeLjRfVFgkz5LIX9vDPbF+PTPa40MMeThhb0/fMMfgKekUu+GPmnwdO1PQnVVGraKrIdN8um4pk1Lk0MpaR0eH3BPLMiwSyd133UXp6f2L0vtipkJbuEdWcOqdt4R96/Fz+Kj2y+ckBwzogQXedntxSfEJHbye0AMLAADAA3QXYEknEfzhnWl9//y8Vyk8bpRqt8udoYPvH3qfoqPFnbGuoqKCVq4Y2APkmXHt9PUoo/O
```

Comment:
looks like this file is identical to  documentation/src/docs/asciidoc/images/connect-prometheus.svg